### ISSUE TITLE
Refused to safe unsafe header

### ISSUE THREAD

**Issue by jb41** [sheetsu/sheetsu-node#4]
I'm getting `Refused to set unsafe header` warning.

Steps to reproduce:
```
var sheetsu = require('sheetsu-node');
sheetsu({ address: '020b2c0f' }).read().then((x) => { console.log(x) })
Refused to set unsafe header "Accept-Encoding"
Refused to set unsafe header "User-Agent"
```

It kind of makes sense, because we don't want to set the User Agent from user's browser.

**Comment by shiffman** [comment]
Hello! I'm getting the same error trying to run a simple example from node. 

```javascript
var sheetsu = require('sheetsu-node');

var config = {
  address: 'ADDRESS',
  api_key: 'KEY',
  api_secret: 'SECRET',
};

var client = sheetsu(config);
client.read({ limit: 2, sheet: "Sheet1" }).then(function(data) {
  console.log(data);
}, function(err){
  console.log(err);
});
```

```
Refused to set unsafe header "Accept-Encoding"
Refused to set unsafe header "User-Agent"
```

**Comment by archibold** [comment]
resolved in 0c6d2c72b76c18e6d77eaaef773199caad5c5873
